libopae/pluginmgr.c

```diff
@@ -73,8 +73,8 @@ static pthread_mutex_t adapter_list_lock =
 
 
 #define MAX_PLUGINS PLUGIN_SUPPORTED_DEVICES_MAX
-static plugin_cfg plugin_list[MAX_PLUGINS];
-static int plugin_count;
+STATIC plugin_cfg *opae_plugin_mgr_plugin_list;
```

Comment:
Should it be called opae_plugin_mgr_config_list so as not to be confused with the adapter table list?